### PR TITLE
Added health-checks to most geoweb containers

### PR DESCRIPTION
**Pull request by Jusaa**
* Added health-checks to most geoweb containers
    * Commented containers with nonexisting version/healthcheck endpoints (publishers & tafplaceholder), those will be uncommented after endpoints are created, also messageservices endpoint to be updated after proper endpoint is created
    * 30s initial wait period for containers with database access & messageconverters
    * 5s initial wait period for other containers
* Added new postgres db container that is deployed by default
    * This allows helm-charts to be deployed without any configuration (fixes pipeline)
    * Only suitable for testing, so for any production environments it can be toggled off by setting db.enableDefaultDb to false
        * It doesn't break anything even if left there, but after setting custom db_secret chart will create empty postgres container
* Added missing required maintainers to Chart.yaml files
* Bumped versions to latest (fe, presets-be, opmet-be)

How to test:
* Pipeline succeeds (meaning that all charts can be installed without any configuration)
* Code review

### CODE REVIEW

File: charts/geoweb-opmet-backend/values.yaml [hunk 1]
```diff
@@ -49,6 +78,20 @@ opmet:
...
+#      initialDelaySeconds: 5
+#      periodSeconds: 5
+#      failureThreshold: 5
+#    readinessProbe:
+#      httpGet:
+#        path: /healthcheck?readiness=true
+#        port: 8090
+#      initialDelaySeconds: 5
+#      periodSeconds: 5
+#      failureThreshold: 1
```
Comment (PauliSairanen) [inline charts/geoweb-opmet-backend/values.yaml:94]: Why is this commented out?

File: charts/geoweb-taf-backend/values.yaml [hunk 2]
```diff
@@ -47,6 +76,20 @@ taf:
...
+#      initialDelaySeconds: 5
+#      periodSeconds: 5
+#      failureThreshold: 5
+#    readinessProbe:
+#      httpGet:
+#        path: /healthcheck?readiness=true
+#        port: 8090
+#      initialDelaySeconds: 5
+#      periodSeconds: 5
+#      failureThreshold: 1
```
Comment (PauliSairanen) [inline charts/geoweb-taf-backend/values.yaml:92]: why is this commented out?

File: charts/geoweb-taf-backend/values.yaml [hunk 3]
```diff
@@ -73,6 +130,28 @@ taf:
...
+#      initialDelaySeconds: 5
+#      periodSeconds: 5
+#      failureThreshold: 5
+#    readinessProbe:
+#      httpGet:
+#        path: /health_check?readiness=true
+#        port: 8085
+#      initialDelaySeconds: 5
+#      periodSeconds: 5
+#      failureThreshold: 1
```
Comment (PauliSairanen) [inline charts/geoweb-taf-backend/values.yaml:146]: why is this commented out?

**Review by PauliSairanen — COMMENTED** [review]
Some commented out code. should we remove these or not?

File: charts/geoweb-opmet-backend/values.yaml (hunk 1 above)
Comment (Jusaa) [reply charts/geoweb-opmet-backend/values.yaml:94]: Publisher container doesn't have any healthcheck endpoint available as of now, so can't implement the liveness/readinessProbes yet, but added the support for it so default probes can be easily enabled afterwards by removing the comments.

File: charts/geoweb-taf-backend/values.yaml (hunk 2 above)
Comment (Jusaa) [reply charts/geoweb-taf-backend/values.yaml:92]: Same as above

File: charts/geoweb-taf-backend/values.yaml (hunk 3 above)
Comment (Jusaa) [reply charts/geoweb-taf-backend/values.yaml:146]: Same situation for tafplaceholder as above publishers

File: charts/geoweb-opmet-backend/values.yaml (hunk 1 above)
Comment (Jusaa) [reply charts/geoweb-opmet-backend/values.yaml:94]: There's user story for creating the missing endpoints: https://gitlab.com/opengeoweb/geoweb-assets/-/issues/2852

**Review by PauliSairanen — APPROVED** [review]

**Pull request merged**